overlord/auth/auth.go

```diff
@@ -28,14 +28,28 @@ import (
 
 	"gopkg.in/macaroon.v1"
 
+	"github.com/snapcore/snapd/asserts"
 	"github.com/snapcore/snapd/overlord/state"
 	"github.com/snapcore/snapd/store"
 )
 
 // AuthState represents current authenticated users as tracked in state
 type AuthState struct {
-	LastID int         `json:"last-id"`
-	Users  []UserState `json:"users"`
+	LastID int          `json:"last-id"`
+	Users  []UserState  `json:"users"`
+	Device *DeviceState `json:"device,omitempty"`
+}
+
+// DeviceState represents the device's identity and store credentials
+type DeviceState struct {
+	Brand  string `json:"brand,omitempty"`
+	Model  string `json:"model,omitempty"`
+	Serial string `json:"serial,omitempty"`
+	// XXX: SerialAssertion should eventually be retrieved from the
+	// assertions DB, not persisted in state.
```

Comment:
I take that back. We probably need those details precisely so we can look up the serial assertion in the database. The alternative would be to look up the single serial assertion, but then we need to constrain the database to contain a single one, which would be awkward. Seems fine to carry the fields here.